tgui/packages/tgui/interfaces/Particool.js

```diff
@@ -0,0 +1,670 @@
+/**
```

Comment:
Do you think you can make this a tsx instead

Reply:
no